src/commonMain/kotlin/baaahs/show/live/OpenPatch.kt

```diff
@@ -37,6 +37,13 @@ class OpenPatch(
             }
         }
 
+    val inEqualsOut: Boolean
```

Comment:
Any insight into why `isFilter` seemed to be behaving badly?

Reply:
So isFilter might be fine, this was before I tried to fetch controls from the openlayout, which doesn't include the "backdrop" control group.

Reply:
Ah okay, I bet it doesn't traverse into children.